subsys/testsuite/boards/unit_testing/unit_testing/Kconfig.defconfig

```diff
@@ -2,4 +2,7 @@
 #
 # Copyright (c) 2022 Nordic Semiconductor
 
-# Intentionally left empty
+# Bluetooth unit tests expect this backend
+choice BT_HCI_BUS_TYPE
+	default BT_RPMSG
```

Comment:
shouldn't that be `BT_NO_DRIVER`?

Reply:
Maybe conceptually, but the current default for `unit_testing` is `RPMSG`, and several tests depend on that symbols `select BT_HAS_HCI_VS`.

Reply:
Sure hope it doesn't open a can of worms

Reply:
Thanks! Ping me if it does.